opm/parser/eclipse/EclipseState/Schedule/Completion.cpp

```diff
@@ -45,7 +46,21 @@ namespace Opm {
         m_connectionTransmissibilityFactor(oldCompletion->getConnectionTransmissibilityFactorAsValueObject()),
         m_skinFactor(oldCompletion->getSkinFactorAsValueObject()),
         m_state(newStatus),
-        m_direction(oldCompletion->getDirection())
+        m_direction(oldCompletion->getDirection()),
+        m_wellPi(oldCompletion->getWellPiAsValueObject())
+    {}
+
+    Completion::Completion(std::shared_ptr<const Completion> oldCompletion, double wellPi)
+            :
+            m_i(oldCompletion->getI()),
+            m_j(oldCompletion->getJ()),
+            m_k(oldCompletion->getK()),
+            m_diameter(oldCompletion->getDiameterAsValueObject()),
+            m_connectionTransmissibilityFactor(oldCompletion->getConnectionTransmissibilityFactorAsValueObject()),
+            m_skinFactor(oldCompletion->getSkinFactorAsValueObject()),
+            m_state(oldCompletion->getState()),
+            m_direction(oldCompletion->getDirection()),
+            m_wellPi("WellPi",wellPi)
```

Comment:
The `wellPi` should be cumulative - i.e. we should multiply with the current value.